.coveragerc

```diff
@@ -8,11 +8,13 @@ omit =
     # TODO(rhcarvalho): this is used to ignore test files from coverage report.
     # We can make this less generic when we stick with a single test pattern in
     # the repo.
-    test_*.py
-    *_tests.py
+    conftest.py
+    */conftest.py
```

Comment:
For some reason on my system it wasn't matching and I had to explicitly add */ to get the files in subdirectories to match.